lib/active_record/tasks/chronomodel_database_tasks.rb

```diff
@@ -50,6 +50,23 @@ def chronomodel_configuration
         @chronomodel_configuration ||= @configuration_hash
       end
 
+      # TODO: replace `run_cmd_with_compatibility` with `run_cmd` and remove when dropping Rails < 8.1 support
+      # Compatibility method to handle Rails version differences in run_cmd signature
+      # Rails < edge: run_cmd(cmd, args, action)
+      # Rails >= edge: run_cmd(cmd, *args, **opts)
+      def run_cmd_with_compatibility(cmd, args, action_description)
+        # Check if run_cmd method accepts keyword arguments (new signature)
+        method_obj = method(:run_cmd)
+        method_parameters = method_obj.parameters
+
+        # If the method accepts rest args (*args), it's the new signature
+        if method_parameters.any? { |type, _name| type == :rest }
```

Comment:
Can we have a single line with a better and short commit?

```suggestion
        # Check if run_cmd method accepts keyword arguments (new signature)
        if method(:run_cmd).parameters.any? { |type, _name| type == :rest }
```